.github/workflows/release.yml

```diff
@@ -22,5 +22,7 @@ jobs:
       - uses: nearform/optic-release-automation-action@v3
         with:
           semver: ${{ github.event.inputs.semver }}
-          commit-message: 'chore: release {version}'
+          commit-message: ‘chore: release {version}’
```

Comment:
these quotes don't seem right, maybe they'll work anyway but I think it's just safer to revert to the standard single quotes